fs2-google-pubsub-http/src/main/scala/com/permutive/pubsub/producer/http/internal/DefaultHttpPublisher.scala

```diff
@@ -34,7 +32,8 @@ private[http] class DefaultHttpPublisher[F[_]: Logger, A: MessageEncoder] privat
     with Http4sClientDsl[F] {
   import DefaultHttpPublisher._
 
-  final private[this] val publishRoute = baseApiUrl.copy(path = baseApiUrl.path.concat(":publish"))
+  final private[this] val publishRoute =
+    Uri.unsafeFromString(s"${baseApiUrl.renderString}:publish")
```

Comment:
`concat` now takes `Uri.Segment` which then concats in a "smart" way, ending up with `path/:foo`, which is *not* what we want.